go/service/pgp.go

```diff
@@ -162,7 +162,7 @@ func (h *PGPHandler) PGPImport(_ context.Context, arg keybase1.PGPImportArg) err
 		LogUI:     h.getLogUI(arg.SessionID),
 		SessionID: arg.SessionID,
 	}
-	eng, err := engine.NewPGPKeyImportEngineFromBytes(arg.Key, arg.PushSecret, nil)
+	eng, err := engine.NewPGPKeyImportEngineFromBytes(arg.Key, arg.PushSecret, h.G())
```

Comment:
The final commit on warning.go says "don't contextualize", so there must have been a reason...

https://github.com/keybase/client/commit/a813b5d14642cfcfe957c95d422abd5798800125#diff-626dc7bce8452b387b5dbfc56efd261aL38